### DISCUSSION TITLE
Explore page side bar form configuration for publisher

### DISCUSSION THREAD

**Discussion by sachinsahu731** [Sunbird-Ed/Community#171] [General Discussion]
In all tabs we configure the organization fields in the sidebar dropdown filter which is working fine for sort publisher .
We are facing below issue -: 
Can we change the label using form config
Can we arrange the order using config
After select the value is not selected 
Form request payload is

`{
    "request": {
    "type": "contentCategory",
    "subType": "global",
    "action": "menubar"
    }
}`

![explore-search (1)](https://user-images.githubusercontent.com/29474792/190607909-7a7494f8-fe37-468f-b716-5e7a148b7de6.png)

**Comment by rajeevsathish** [comment]
@sachinsahu731 we are not able to understand the Question more clearly can you be more elaborate so that we can understand your need and help you regarding the Requirement

**Comment by sachinsahu731** [comment]
@rajeevsathish  Our goal is to enable publisher dropdown in the sidebar filter for all tabs . Where users can filter the content card based on the publisher .
For this we  are updating the form config `{ "request": { "type": "contentCategory", "subType": "global", "action": "menubar" } }`
Inside the all tab object ` search.facets` we added organization property .
After updating the Form the publisher filter is enabled and it sorts the card based on publisher .
But we faced the below issue -:

- Currently the label is an organization. Can we change the label using form config?

- The Organization is showing in the First level . Can we rearrange this ?

-  After selecting the publisher value the card filter sorting is working But the selected value is not showing in the dropdown.

Can we solve the above issue using form config or its required code change.?

**Reply by sachinsahu731** [reply]
@sharathkashyap  @rajeevsathish  any update ?

**Comment by sharathkashyap** [comment]
@sachinsahu731 we have included this issue in a roadmap. Today, we have the following features available as part of configuration.
In All Pages, It is possible to include any supported facet as filter.This is purely a configurational change. There are some gaps which we will try to cover.
`a)  Currently the label is an organization. Can we change the label using form config?`
Yes it is possible to change the label of the filter using form configuration

b) The Organization is showing in the First level . Can we rearrange this ?
Yes it is possible to decide the order of Filter in configuration.

c) After selecting the publisher value the card filter sorting is working But the selected value is not showing in the dropdown.
This Looks like a Bug to us. We have raised the ticket for the same.

**Reply by sachinsahu731** [reply]
@sharathkashyap  Above point a and b is also not possible form configuration . Also can you provide the Jira ticket no where we can track this status.
cc -: @rajeevsathish
